index.html

```diff
@@ -425,7 +425,7 @@ <h2 id="steps">Steps for
 <span class="transwho" data-profile="CR|PR|PR-OBSL|PR-RSCND" data-cr="new|substantive|rec-update|rec-amended">
 	creates a <a href="#transreq">transition request</a> in <a href='https://github.com/w3c/transitions/issues'>w3c/transitions</a>.
 </span>
-<span class="transwho" data-profile="REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">
+<span class="transwho" data-profile="FPWD|REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">
```

Comment:
Actually, we use GH for FPWD.
```suggestion
<span class="transwho" data-profile="REC|OBSL|RSCND" data-cr="new|substantive|rec-update|rec-amended">
```